zos-utils/zossrc/AttlsContext.c

```diff
@@ -322,43 +374,83 @@ jint JNI_OnLoad(JavaVM *vm, void *reserved)
 
     // fetch AtllsContext.class
     jclass clazz = (*env) -> FindClass(env, JNI_CLASS_ATTLS_CONTEXT);
+    if ((*env) -> ExceptionCheck(env)) return JNI_VERSION;
```

Comment:
All these calls are great candidates for a macro:

```suggestion
#define EXCEPTION_CHECK (retcode, call)                 \
    call;                                               \
    if ((*env) -> ExceptionCheck(env)) return retcode;

    EXCEPTION_CHECK_AND_RETURN(JNI_VERSION, jclass clazz = (*env) -> FindClass(env, JNI_CLASS_ATTLS_CONTEXT));

```